test/support/candlepin/owner_support.rb

```diff
@@ -38,6 +38,13 @@ def self.create_organization(name, label)
       @organization
     end
 
+    def self.import_manifest(owner_label, path_to_file)
+      path = "/candlepin/owners/#{owner_label}/imports?force=SIGNATURE_CONFLICT&force=MANIFEST_SAME"
```

Comment:
The manifests generated by the utility mentioned in the tests are not properly signed. Due to that we need to add special params to the candlepin API call. Opted not to add them to the actual app code since this is testing-only.